musicbot/bot.py

```diff
@@ -1140,7 +1141,10 @@ def _gen_embed(self):
         if not self.is_all:
             desc += self.str.get('cmd-help-all', '\nOnly showing commands you can use, for a list of all commands, run `{}help all`').format(prefix)
 
-        return Response(desc, reply=True, delete_after=60)
+        if self.config.dmhelp:
+            await self.safe_send_message(message.author, desc, expire_in=60)
```

Comment:
It feels like we still should give acknowledgement in the user's channel that we've DM'd them?